### PR TITLE
make: Allow to build documentation with podman

### PR DESCRIPTION
**Pull request by vadorovsky**
Use CONTAINER_ENGINE_FULL variable to allow using podman instead of
docker.

Signed-off-by: Michal Rostecki <mrostecki@opensuse.org>

### CODE REVIEW

**Comment by mrostecki** [comment]
test-me-please

**Review by tklauser — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -69,3 +69,6 @@ test/bpf/_results
 
 # generated from make targets
 *.ok
+
+# Documentation
+Documentation/.config/
```
Comment (pchaigno) [inline .gitignore]: Why is this needed? What writes this directory?

**Review by pchaigno — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (vadorovsky) [reply .gitignore]: It's unrelated to the PR itself, but I noticed that this dir gets created after `make -C Documentation`.

**Comment by mrostecki** [comment]
The failure is unrelated

```
12:59:15  • Failure [329.177 seconds]
12:59:15  K8sDatapathConfig
12:59:15  /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/k8s-1.11-gopath/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:395
12:59:15    Encapsulation
12:59:15    /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/k8s-1.11-gopath/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:395
12:59:15      Check connectivity with Geneve encapsulation [It]
12:59:15      /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/k8s-1.11-gopath/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:430
12:59:15  
12:59:15      cilium pre-flight checks failed
[2020-04-14T10:59:15.809Z]     Expected
[2020-04-14T10:59:15.809Z]         <*errors.errorString | 0xc000c94460>: {
[2020-04-14T10:59:15.809Z]             s: "CiliumPreFlightCheck error: Timeout reached: PreflightCheck failed: Last polled error: connectivity health is failing: Cluster connectivity is unhealthy on 'cilium-2jdcv': Exitcode: 255 \nStdout:\n \t \nStderr:\n \t Error: Cannot get status/probe: Put \"http://%2Fvar%2Frun%2Fcilium%2Fhealth.sock/v1beta/status/probe\": context deadline exceeded\n\t \n\t command terminated with exit code 255\n\t \n",
[2020-04-14T10:59:15.809Z]         }
[2020-04-14T10:59:15.809Z]     to be nil
```

I'm only changing docs build here. And I probably did a mistake by triggering all tests...

**Comment by mrostecki** [comment]
test-docs-please

File: .gitignore (hunk 1 above)
Comment (pchaigno) [reply .gitignore]: I don't get that directory after running that command. Maybe something related to your environment? Could that be related to podman?

Comment (aanm) [reply .gitignore]: cc @errordeveloper

**Review by aanm — APPROVED** [review]
LGTM as long Paul's comments are addressed

File: .gitignore (hunk 1 above)
Comment (vadorovsky) [reply .gitignore]: After rebasing with master that file is gone for me, both for docker and podman... weird.

**Comment by mrostecki** [comment]
I removed the .gitignore change, PTAL.

**Comment by mrostecki** [comment]
test-docs-please
